src/components/Navigation/LogoHeader.js

```diff
@@ -10,15 +10,15 @@ const Container = styled.div`
 `;
 
 const Text = styled.span`
-  font-size: ${({ theme }) => theme.header.fontSize};
-  font-weight: ${({ theme }) => theme.header.fontWeight};
+  font-size: 22px;
```

Comment:
This change is a little concerning, moving font information locally to the component will make it more challenging in the future to support responsive designs such as mobile.